pages/infrastructure_providers-validators/running_full_node.md

```diff
@@ -1,73 +1,60 @@
-# Running a Full Node
-
-### Save your Chain ID in `dydxprotocold` config
-
-Save the [chain-id](../infrastructure_providers-network/network_constants.mdx#chain-id). This will make it so you do not have to manually pass in the chain-id flag for every CLI command.
-
-```bash
-dydxprotocold config chain-id $CHAIN_ID
+# Run a Full Node
+Running a full node allows your system to participate in a dYdX chain network. The recommended settings and best practices on this page help ensure your node stays healthy and up to date with the network.
+
+> Code snippets on this page use example values. Replace them with your own. See the [Network Configuration](../infrastructure_providers-network/network_constants.mdx) section of the documentation for network constants and other resources you need to configure a full node.
+
+## Prerequisites
+You need a running, non-validating full node that is connected to network. 
```

Comment:
Kinda a dumb question:

I am confused why having a running, non-validating full node is a prereq for this page. Why is this a prereq? Isnt this page outlining how to run one?

Reply:
Oh after reading the rest of the doc, I think this is about maintaining a running node? If so, should we highlight this more in the top section of this page + consider updating the title?

Reply:
Yeah, this is essentially about best practices for a node you have running already. Can definitely make the title more specific, like "Optimize Your Full Node"  -- from convos with Tommy it sounds like MM care a lot about that (to be on par with competition).